tests/src/Model/Response/JobSummaryCollectionTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenEuropa\Tests\CdtClient\Model\Response;
+
+use OpenEuropa\CdtClient\Model\Response\JobSummaryCollection;
+use OpenEuropa\Tests\CdtClient\Traits\CollectionTestTrait;
+use OpenEuropa\Tests\CdtClient\Traits\ResponseModelTestTrait;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @coversDefaultClass \OpenEuropa\CdtClient\Model\Response\JobSummaryCollection
+ */
+class JobSummaryCollectionTest extends TestCase
```

Comment:
feels a bit repetative to test each child of the base collection, think that a test where a vanilla class extends the base collection and test its methods would have been sufficient

Reply:
I agree, but on the other hand, the vanilla class approach reduces test coverage significantly. I was torn on this one.